packages/flutter_test/lib/src/controller.dart

```diff
@@ -506,6 +517,17 @@ abstract class WidgetController {
   /// of time, consider using [timedDrag].
   ///
   /// {@template flutter.flutter_test.WidgetController.drag}
+  /// The `offset` represents the motion one's finger would perform on the
```

Comment:
I think you can put a template inside of a template, I can't recall it, but I am pretty sure I have done this before. :)

Reply:
How do I deal with the sentences being slightly different?

Reply:
You can probably re-word it to put the duplicates in a template, and leave what is unique out of the template?

Reply:
Hmm, not sure if that is useful / worth the effort

Will think about it when I have time

Reply:
Ok, no worries at all. I will close this PR then. If you feel like coming back and working on it, feel free to re-open. :)